apsi-enrollment-fe/src/app/modules/main/pages/event-detail/event-detail.component.ts

```diff
@@ -11,11 +12,17 @@ import { EventService } from '../../services/event/event.service';
 export class EventDetailComponent implements OnInit {
   show = false;
   event: Event;
+  currency: String;
 
   enrollmentStatusEnum = EnrollmentStatus;
   enrollmentStatus: EnrollmentStatus;
 
-  constructor(private eventService: EventService) {}
+  constructor(
+    private eventService: EventService,
+    private paymentService: PaymentService
```

Comment:
You should add mock syp object in `.spec.ts` file for tests to pass

Reply:
added